x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/findings_flyout.tsx

```diff
@@ -33,10 +35,10 @@ import k8sLogoIcon from '../../../assets/icons/k8s_logo.svg';
 import { ResourceTab } from './resource_tab';
 import { JsonTab } from './json_tab';
 
-const tabs = ['remediation', 'resource', 'general', 'json'] as const;
+const tabs = ['Remediation', 'Resource', 'General', 'JSON'] as const;
```

Comment:
if these are keys for the code, I think lower case is better.
From the casing I'm assuming you are using it as titles in the UI - instead we should use proper keys for localization and map the tab keys to the proper localization key the represents that value

Reply:
You're correct but that was the current implementation, this change was purely to change the default `capitalizing` rule that we had there in css. I think the change that you are suggesting should be done on another PR since it involves more than style changes. wdyt? do you prefer to do that here anyway?

Reply:
yes :) ty